### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by vijayank88**
Updated ORFS readme file.

### CODE REVIEW

**Comment by maliberty** [comment]
We already give a link to the documentation that describes how to setup ORFS.  Why are we repeating that here?

**Comment by vijayank88** [comment]
Previously main readme don't have much details.
So added in unique format of what other repo has such as keep installation instructions at main page, so easy to follow up for beginners.

**Comment by vijayank88** [comment]
> We already give a link to the documentation that describes how to setup ORFS. Why are we repeating that here?

@maliberty 
Now just pointing installation markdown file. Removed steps from `README.md` file.

**Comment by vijayank88** [comment]
cc/ @dralabeing

File: docs/images/ORFS_Flow.webp
Comment (vvbandeira) [inline docs/images/ORFS_Flow.webp:1]: Would a vertical image help visualization?

File: README.md [hunk 1]
```diff
@@ -3,16 +3,61 @@
...
 [![Docs](https://readthedocs.org/projects/openroad-flow-scripts/badge/?version=latest)](https://openroad-flow-scripts.readthedocs.io/en/latest/?badge=latest)
 
-OpenROAD Flow is a full RTL-to-GDS flow built entirely on open-source tools.
-The project aims for automated, no-human-in-the-loop digital circuit design
-with 24-hour turnaround time.
+OpenROAD-flow-scripts(ORFS) is a fully autonomous, RTL-GDSII flow
+for rapid architecture and design space exploration, early prediction
+of QoR and detailed physical design implementation. However,  ORFS
+also enables manual intervention for finer user control of individual
+flow stages through tcl commands and python APIs.
```
Comment (vvbandeira) [inline README.md]:
```suggestion
OpenROAD-flow-scripts (ORFS) is a fully autonomous, RTL-GDSII flow
for rapid architecture and design space exploration, early prediction
of QoR and detailed physical design implementation. However, ORFS
also enables manual intervention for finer user control of individual
flow stages through Tcl commands and Python APIs.
```

File: README.md [hunk 2]
```diff
@@ -3,16 +3,61 @@
...
+### Docker Based Installation
+
+To ease dependency installation issues, ORFS uses docker images.
+Docker image includes ORFS binaries, applications as well as all
+required dependencies. All of the flow tools are encapsulated
+inside the container image.
+
+If `Docker` is not installed already, refer to the document
+[here](./docs/user/DockerInstall.md) on how to install and how to
+manage docker as non-root user.
```
Comment (vvbandeira) [inline README.md]: I would prefer to just point to the official Docker install page and not add the `DockerInstall.md` file -- which already has misleading information, as there's no need to reboot the system for the `usermod` changes to take effect; only opening a new shell is enough.

File: README.md [hunk 3]
```diff
@@ -3,16 +3,61 @@
...
+```
+
+Above steps run from RTL-GDSII generation and you can view final
+layout with OpenROAD GUI as,
+
+```
+make gui_final
+```
+
+![gcd_final.webp](./docs/images/gcd_final.webp)
```
Comment (vvbandeira) [inline README.md]: verifying the installation should be in the same file as the install instructions, as they will be different between local and docker installs.

File: README.md [hunk 4]
```diff
@@ -3,16 +3,61 @@
...
 
 ## Using the Flow
 
 - See the OpenROAD [documentation here](https://openroad.readthedocs.io/en/latest/).
 - How to [start using OpenROAD flow here](https://openroad-flow-scripts.readthedocs.io/en/latest/user/GettingStarted.html).
-- Our [user guide here](https://openroad-flow-scripts.readthedocs.io/en/latest/user/UserGuide.html).
-- Our [Flow Tutorial here](https://openroad-flow-scripts.readthedocs.io/en/latest/tutorials/FlowTutorial.html).
+- See ORFS [user guide here](https://openroad-flow-scripts.readthedocs.io/en/latest/user/UserGuide.html).
+- See ORFS [Flow Tutorial here](https://openroad-flow-scripts.readthedocs.io/en/latest/tutorials/FlowTutorial.html).
```
Comment (vvbandeira) [inline README.md]: We should probably rewrite this list to include a short description of what the linked document provides. Example: "For details about the individual flow commands, see OR docs [link]. For details about the automated flow, see ORFS docs [link]" ....

File: docs/user/DockerInstall.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+as non-root user to avoid permission issues while running the flow.
+Run below commands to make docker as non-root user.
+
+```
+sudo groupadd docker
+sudo usermod -aG docker $USER
+sudo reboot # REBOOT!
+```
+
+You must restart your operating system for the group permissions to apply.
```
Comment (vvbandeira) [inline docs/user/DockerInstall.md]: As mentioned above, this is not true. The user needs to open a new shell or log off and log in again.

**Review by vvbandeira — CHANGES_REQUESTED** [review]

File: docs/user/BuildLocally.md [hunk 6]
```diff
@@ -43,6 +43,37 @@ make gui_final
 
 ![gcd_final.webp](../images/gcd_final.webp)
 
+## Using Pre-built Binaries
+
+You can download pre-built binaries with self contained dependencies
+included from the Precision Innovations github repository
```
Comment (tspyrou) [inline docs/user/BuildLocally.md:52]: Please add verbiage like this to make sure its known that PII is providing this binary as a service to the community vs it coming from UCSD or the project.

"Thanks to Precision Innovations (www.precisioninno.com) for providing and supporting this daily release of prebuilt installers to the community."

**Review by tspyrou — CHANGES_REQUESTED** [review]

File: docs/user/BuildLocally.md (hunk 6 above)
Comment (vvbandeira) [reply docs/user/BuildLocally.md:52]: Added at the end of the section.

**Review by vvbandeira — APPROVED** [review]

**Pull request merged**